mlflow/cli.py

```diff
@@ -144,6 +144,20 @@ def run(uri, entry_point, version, param_list, experiment_name, experiment_id, m
         _logger.error("=== %s ===", e)
         sys.exit(1)
 
+def _validate_server_args(static_prefix=None, gunicorn_opts=None, workers=None, waitress_opts=None):
+    server_kwargs = {'static_prefix': static_prefix}
```

Comment:
is static_prefix understood only by gunicorn?

Reply:
@tomasatdatabricks my limited understanding is that this is a flask routing concern and not the actual webserver (and a quick search around the codebase appears to reflect that) , but I'm certainly out of my depth here - happy to learn how it gets leveraged by gunicorn

static_prefix -> env_map[STATIC_PREFIX_ENV_VAR] -> _add_static_prefix -> @app.route(_add_static_prefix('/get-artifact'))

Reply:
I see, that makes sense, thanks for the clarification!